db/schema_applier.cc

```diff
@@ -126,7 +126,7 @@ static std::optional<table_id> table_id_from_mutations(const schema_mutations& s
     if (table_rs.empty()) {
         return std::nullopt;
     }
-    query::result_set_row table_row = table_rs.row(0);
+    const query::result_set_row& table_row = table_rs.row(0);
     return table_id(table_row.get_nonnull<utils::UUID>("id"));
```

Comment:
```suggestion
    return table_id(table_rs.row(0).get_nonnull<utils::UUID>("id"));
```

Reply:
this is not in the scope of this change which prevents copies of result_set_row. how about creating a follow up to address your proposal?

Reply:
It is just a nitpick, not important in any way.
I am OK with not accepting it.